hello.go

```diff
@@ -86,13 +86,24 @@ func main() {
 
 	http.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
 		tmpl.Execute(w, data)
+		log.Printf("%s request from %s for %s", r.Method, r.Host, r.URL)
+	})
+
+	http.HandleFunc("/cloud_bg.svg", func(w http.ResponseWriter, r *http.Request) {
+		log.Printf("%s request for %s from %s.", r.Method, r.Host, r.URL)
+		http.ServeFile(w, r, "cloud_bg.svg")
+	})
+
+	http.HandleFunc("/lightbulb_icon.svg", func(w http.ResponseWriter, r *http.Request) {
+		log.Printf("%s request for %s from %s.", r.Method, r.Host, r.URL)
+		http.ServeFile(w, r, "lightbulb_icon.svg")
 	})
```

Comment:
If you implement the request above of moving to `assets/`, we can do something much simpler, cleaner and future-proof, like:

    http.Handle("/assets", http.FileServer(http.Dir("./assets")))

Reply:
Done.